examples/QmlBarcodeReader/cmake/Locations.cmake

```diff
@@ -0,0 +1,37 @@
+set(COMPANY "Scythe Studio")
+set(COPYRIGHT "Copyright (c) 2022 Scythe Studio. All rights reserved.")
+set(IDENTIFIER "com.scythestudio.scodes.example")
+
+# ---CONFIGURATION---
+option(USE_QML "Add QML support" ON)
+option(USE_LIBS "Use external libraries" ON)
+
+# Locations - directories in project structure
+set(LIB_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../src)
```

Comment:
Instead of mixing relative paths and variables you can use CMAKE_SOURCE_DIR variable - this points to directory containing the root CMakeLists.txt, which in that case is a directory of QmlBarcodeReader.